### PR TITLE
Change phrasing "peak" to "Historical maximum"

### PR DESCRIPTION
**Pull request by akh4rf**
## Motivation

Updating phrasing on certain components: #797 

## Changes

image

### CODE REVIEW

File: modules/speed/constants/speeds.ts [hunk 1]
```diff
@@ -77,11 +76,12 @@ export const CORE_TRACK_LENGTHS = {
...
 
-export const PEAK_MPH = {
-  'line-red': CORE_TRACK_LENGTHS['line-red'] / (PEAK_COMPLETE_TRIP_TIMES['line-red'].value / 3600),
-  'line-orange':
-    CORE_TRACK_LENGTHS['line-orange'] / (PEAK_COMPLETE_TRIP_TIMES['line-orange'].value / 3600),
-  'line-blue':
-    CORE_TRACK_LENGTHS['line-blue'] / (PEAK_COMPLETE_TRIP_TIMES['line-blue'].value / 3600),
-  DEFAULT: 1,
-};
+// Commented this out because these resulted in values slightly higher than the actual peaks in our data
```
Comment (devinmatte) [inline modules/speed/constants/speeds.ts:79]: I'd prefer not to leave blocks of commented code in our codebase. If you prefer to leave a reference you could leave a comment like
```
// Peak_MPH removed in #825
```

**Review by devinmatte — COMMENTED** [review]
Looks like lint is failing due to prettier issues. 

If you're using vscode, it should be able to autoformat for you using eslint. For other editors/cli run Eslint directly and they should be all autofixable issues

**Review by devinmatte — APPROVED** [review]
Looks good to me, but going to recommend waiting on @PatrickCleary for the final say so

**Comment by PatrickCleary** [comment]
Yes, also looks good to me. I was looking at it though, and saw that we were still using "Peak" on the descriptions of the landing page charts. So I wanted to rework those a bit, which I'm currently working on.  Will push a PR for that shortly and we can merge these together

**Comment by akh4rf** [comment]
Got it, let me know when you're ready @PatrickCleary

**Review by PatrickCleary — APPROVED** [review]

**Pull request merged**